.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---

```

Comment:
A cool feature of the https://github.com/architect/.github/tree/main/.github folder is that you don't need to copy these files into the repo. You can but by using https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file we can limit these files to one repo to make updating easier.

Reply:
Oh right! thanks for the reminder. I will trim them down before merging. thanks